lib/internal/Magento/Framework/Stdlib/DateTime/Timezone.php

```diff
@@ -5,11 +5,18 @@
  */
 namespace Magento\Framework\Stdlib\DateTime;
 
-use \Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Framework\App\ScopeInterface;
+use Magento\Framework\App\ScopeResolverInterface;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Locale\ResolverInterface;
 use Magento\Framework\Phrase;
+use Magento\Framework\Stdlib\DateTime;
```

Comment:
Please, add an alias for DateTime or use FQCN due to Magento 2.2.x support PHP 7.0 version and this code will fail with fatal error:

```
Cannot use Magento\Framework\Stdlib\DateTime as DateTime because the name is already in use in /opt/bamboo/builds/MPAF-SPERFEE-CEEC/build-775/lib/internal/Magento/Framework/Stdlib/DateTime/Timezone.php on line 14

```

This is a PHP [core bug](https://bugs.php.net/bug.php?id=66862), that was fixed in [7.0.13](http://php.net/ChangeLog-7.php#7.0.13)

Reply:
Thanks for that. I wasn't aware of that bug. Is this 2.2.x specific, as the same commit was made to the 2.3-develop branch.